Lib/posixpath.py

```diff
@@ -500,7 +500,7 @@ def commonpath(paths):
         split_paths = [path.split(sep) for path in paths]
 
         try:
-            isabs, = set(p[:1] == sep for p in paths)
+            isabs, = {p[:1] == sep for p in paths}
```

Comment:
Perhaps. However, my intention with this change was to avoid logic changes. I'll revert `posixpath.py` and `ntpath.py` for now.